src/wirecloud/platform/workspace/crud.py

```diff
@@ -105,12 +105,12 @@ async def create_workspace(db: DBSession, request: Request, owner: UserAll, mash
     if type(mashup) == str:
         values = mashup.split('/', 3)
         if len(values) != 3:
-            raise ValueError('Invalid mashup id')
+            raise ValueError(_('Invalid mashup id'))
 
         (mashup_vendor, mashup_name, mashup_version) = values
         resource = await get_catalogue_resource(db, mashup_vendor, mashup_name, mashup_version)
         if resource is None or not await resource.is_available_for(db, owner) or resource.resource_type() != 'mashup':
-            raise ValueError(f"Mashup not found {mashup}")
+            raise ValueError(_(f"Mashup not found {mashup}"))
```

Comment:
No se puede hacer una traducción después de reemplazar. Tendría que ser:
```suggestion
            raise ValueError(_(f"Mashup not found %(mashup)s") % {"mashup": mashup})
```